access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java

```diff
@@ -304,7 +304,7 @@ public SearchResultResponse getFacetList(SearchRequest searchRequest) {
 	 *           Access groups of the user making this request.
 	 * @return
 	 */
-	public List<BriefObjectMetadataBean> getNeighboringItems(BriefObjectMetadataBean metadata, int windowSize,
+	public List<BriefObjectMetadataBean> getNeighboringItems(BriefObjectMetadata metadata, int windowSize,
```

Comment:
just curious why you made this change. do you envision scenarios where a different type of BriefObjectMetadata might be used?

Reply:
Generally I'd prefer to pass in the interface when possible. There is actually another implementation of BriefObjectMetadata. I ended up reverting this though because I needed to add the record to the result list, which needed to be the implementation.